### PR TITLE
ENH: Flexibilize views of ``plot_mosaic`` to render nonhuman imaging

### LINKED ISSUE #1: Add logic elements to allow switching reportlets into "rodent-mode"
As per meeting today.

### PR DESCRIPTION
**Pull request by oesteban**
Makes ``plot_mosaic`` more configurable to decide what view(s) are to be plotted, and which one will be the main view.

Related: nipreps/mriqc#1027.
Resolves: #1.

### CODE REVIEW

**Comment by oesteban** [comment]
@eilidhmacnicol https://output.circle-artifacts.com/output/job/0d03946e-489f-4fd5-9237-e579909ac91a/artifacts/0/tmp/tests/output/rodent_mosaic.svg

**Comment by eilidhmacnicol** [comment]
@oesteban plot looks beautiful.

I started reviewing the changes yesterday but see you've added more today. Do you want me to hold off until you're happy or are you happy for me to review as you go?

**Comment by oesteban** [comment]
Yes, yesterday I was still playing with the relative size depending on the number and which views you want to plot. But I'm done with that part, so the code should be reviewable right now.

File: nireports/interfaces/mosaic.py [hunk 1]
```diff
@@ -112,6 +112,29 @@ def _run_interface(self, runtime):
...
+        usedefault=True,
+    )
+    addon_view2 = traits.Enum(
+        None,
+        "axial",
+        "sagittal",
+        "coronal",
+        default=None,
+        usedefault=True,
+    )
```
Comment (effigies) [inline nireports/interfaces/mosaic.py:137]: Would it make sense to make this
```suggestion
    view = traits.List(
        traits.Enum("axial", "sagittal", "coronal"),
        default=["axial", "sagittal"],
        help="Sequence of views to plot (up to three)",
        usedefault=True,
    )
```

I think there's minLength/maxLength or something as well.

**Comment by effigies** [comment]
Sorry for the drive-by. Just looking at the interface, not attempting an in-depth review at this time.

**Comment by oesteban** [comment]
Thanks Chris, I think the interface is more usable now.

@eilidhmacnicol I'm trying to wrap your PR for MRIQC in the next release. So I will hold on just a little, and merge. If you have time before that great -- if not, please open an issue and I'll address whatever problem you find :).

**Pull request merged**